Buildings/Fluid/AirFilters/Empirical.mo

```diff
@@ -283,43 +256,51 @@ Documentation(info="<html>
 <p>
 An empirical model of air filters, which considers the impacts of the contaminant
 accumulation on the pressure drop and the filtration efficiency.
-The dynamic characteristics of the filters are defined by three
-parameters, <code>mCon_nominal</code>,<code>epsFun</code>, and <code>b</code>.
+The dynamic characteristics of the filters are defined by the following parameters.
 </p>
 <ul>
 <li>
 The <code>mCon_nominal</code> determines the maximum mass of the contaminants that the
-filter can capture.
+filter can hold.
 </li>
 <li>
-The <code>epsFun</code> is a vector of coefficients that determines how the
-filtration efficiency changes with the contaminant accumulation.
+A performance dataset <code>per</code> that determines 
+<ul>
+<li>
+how the flow coefficient changes with the contaminant accumulation;
 </li>
 <li>
-The <code>b</code> is a constant that determines how the flow coefficient changes
-with the contaminant accumulation.
+the types of contaminants;
+</li>
+<li>
+how the filtration efficiency changes 
+with the contaminant accumulation for each contaminant.
+</li>
+</ul>
 </li>
 </ul>
 <p>
 See more detailed descriptions in
-<a href=\"modelica://Buildings.Fluid.AirFilters.BaseClasses.FiltrationEfficiency\">
-Buildings.Fluid.AirFilters.BaseClasses.FiltrationEfficiency</a> and
 <a href=\"modelica://Buildings.Fluid.AirFilters.BaseClasses.FlowCoefficientCorrection\">
-Buildings.Fluid.AirFilters.BaseClasses.FlowCoefficientCorrection</a>.
+Buildings.Fluid.AirFilters.BaseClasses.FlowCoefficientCorrection</a>
+and <a href=\"modelica://Buildings.Fluid.AirFilters.BaseClasses.FiltrationEfficiency\">
+Buildings.Fluid.AirFilters.BaseClasses.FiltrationEfficiency</a>, respectively.
 </p>
 <p>
-The input boolean flag, <code>uRep</code>, triggers the filter replacement.
-When <code>uRep</code> changes from <code>false</code> to <code>true</code>, the
-mass of the captured contaminant becomes <i>0</i>.
+The input boolean flag, <code>uRep</code>, triggers the filter replacement, i.e.,
+when <code>uRep</code> changes from <code>false</code> to <code>true</code>, the
+mass of the captured contaminants becomes <i>0</i>.
 </p>
 <b>Note:</b>
+Warnings will be triggered 
 <ul>
 <li>
-A warning will be triggered when the captured contaminant mass becomes greater than the
-maximum contaminant mass (<code>mCon_nominal</code>).
+when the captured contaminant mass becomes greater than the
+<code>mCon_nominal</code>.
 </li>
 <li>
-The <code>extraPropertiesNames</code> has to be defined in the medium model.
+when the <code>extraPropertiesNames</code> in the medium model don't contain all the contaminants
+defined in the <code>per</code>.
 </li>
 </ul>
 </html>", revisions="<html>
```

Comment:
Do we reference for the model development?

Reply:
We don't have a reference.